manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ElementTechnologie.openElement
+PackageVersion: 1.57.9
+InstallerType: portable
+Installers:
+- Architecture: x86
+  InstallerUrl: https://openelement.uk/Files/Other/openElement.exe
+  InstallerSha256: 22F33F86128B59D3911702150EDA6438C2275065C765EE550491A601FF568AE6

```

```suggestion
  InstallerSha256: 22F33F86128B59D3911702150EDA6438C2275065C765EE550491A601FF568AE6
  InstallerLocale: en-US
  InstallerSwitches:
    Custom: /exelang 1033 /exenoui /qn
    InstallLocation: '/extract:<INSTALLPATH>'
    Log: /exelog <LOGPATH>
```

- This package contains both English (language ID `1033`) and French (language ID `1036`) profile.
- `InstallerSwitches` can be found by adding `/?` switch to the installer file.